src/dcos_e2e/cluster.py

```diff
@@ -90,6 +90,36 @@ def from_nodes(
             cluster_backend=backend,
         )
 
+    @retry(
+        exceptions=(subprocess.CalledProcessError),
+        tries=500,
+        delay=5,
+    )
+    def _wait_for_dcos_diagnostics(
+        self,
+        log_output_live: bool = False,
```

Comment:
There are no users of `log_output_live` and this is private. Please either remove it as an option or make it possible to pass through from `wait_for_dcos` methods and add tests for that.